source/global-this.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+Aids in defining global symbols in a narrow scope. Permits extending `globalThis`.
```

Comment:
The first line here should be the same as in the readme. Below this first line could be a more in-depth explanation.

Reply:
Done.